src/platform/silabs/rs911x/BLEManagerImpl.cpp

```diff
@@ -114,9 +117,6 @@ void sl_ble_event_handling_task(void)
     while (1)
     {
         //! This semaphore is waiting for next ble event task
```

Comment:
move the comment where. you moved the `rsi_semaphore_wait` call.

Also why is it moved only to the default? Can there be a ble event without the semaphore being posted first?

Reply:
@jmartinez-silabs added this in JIRA i'll check and update this in upcoming PR
https://jira.silabs.com/browse/MATTER-2184